unicodetools/src/main/java/org/unicode/text/UCD/TestUnicodeInvariants.java

```diff
@@ -113,7 +116,31 @@ enum Expected {empty, not_empty, irrelevant};
 
     private static PrintWriter out;
 
-    public static void testInvariants(String outputFile, boolean doRange) throws IOException {
+    /**
+     * Fetch a reader for our input data.
+     * @param inputFile if null, read DEFAULT_FILE from classpath
```

Comment:
Why would it be null? Line 80 calls testInvariants() with DEFAULT_FILE, or with a String from a command-line argument, if there is one.

Reply:
I pass null from TestTestUnicodeInvariants to mean default

Reply:
How about passing TestUnicodeInvariants.DEFAULT_FILE from the TestTest... code?